src/lib/components/Ideogram.react.js

```diff
@@ -477,15 +472,23 @@ Ideogram.propTypes = {
     dataDir: PropTypes.string,
 
     /**
-     * This is a work in progess and will hopefully be fixed in future releases.
+     * Organism(s) to show chromosomes for. Supply organism's name as a string (e.g. "human") or
+     * organism's NCBI Taxonomy ID (taxid, e.g. 9606) to display chromosomes from a single organism,
```

Comment:
```suggestion
     * organism's NCBI Taxonomy ID (taxID, e.g., 9606) to display chromosomes from a single organism,
```
Just because I'm seeing "taxID" later on.